ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-service-insertion.j2

```diff
@@ -7,4 +7,28 @@
 {% if router_service_insertion.enabled is arista.avd.defined(true) %}
 !
 router service-insertion
+{%     if router_service_insertion.connections is arista.avd.defined %}
+{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') %}
```

Comment:
May be..
```suggestion
{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') | arista.avd.default([]) %}
```
@guillaumeVilar

Reply:
wrong Guillaume @Vibhu-gslab  ;)

otherwise yes - remove line 10 and replace it as Vibhu mentions using default

Reply:
Fixed